etc/tutorial/cylc-forecasting-suite/lib/python/util.py

```diff
@@ -265,7 +267,7 @@ def __call__(self, grid_x, grid_y):
 
 
 def parse_domain(domain):
-    bbox = map(float, domain.split(','))
+    bbox = list(map(float, domain.split(',')))
```

Comment:
is this `list` necessary?

Reply:
Yes, because of the index access on the following lines. Alternatively it could be a one-liner:

```python
return dict(zip(['lng1', 'lat1', 'lng2', 'lat2']), map(float, domain.split(','))
```